src/styles/blocks/_main.scss

```diff
@@ -0,0 +1,6 @@
+.main {
+  &__catalog {
+    margin: 0 auto;
+    padding: 50px $catalog-side-padding 72px;
+  }
+}
```

Comment:
i like that you remember about additional element for positioning and used it. i forgot about it, thanks :)